### DISCUSSION TITLE
Ever Considered PVP Server?

### DISCUSSION THREAD

**Discussion by lanceroy** [chromiecraft/chromiecraft#2453] [General]
So far been really enjoying the server. Thanks for being an awesome team!

I'm sure this has been brought up before, but has there ever been considerations on a PVP version on the server?

**Comment by 55Honey** [comment]
Hi lanceroy,

Thanks for the feedback, we appreciate it.
Indeed that has been brought up quite a few times. Tl;dr: ChromieCraft will **never** change it's ruleset to  PvP.
Also, a second realm is completely out of our scope for now.

**Reply by anthonygauthier** [reply]
Hi @55Honey, would it be possible to get the reason behind not wanting to go for a PvP realm (quick search on GitHub didn't seem to do the trick haha)? I'm just curious as to why; is it an "owner's preference" or to avoid players trolling, etc.

Thanks! :smile:

**Reply by 55Honey** [reply]
The subject has been discussed in-depth on our discord server. We keep suggesting people to join it for more and better community interaction.

https://discord.gg/jUdJRhmT3J or
https://discord.gg/ChromieCraft 

There you go :-)

**Reply by anthonygauthier** [reply]
Oh yeah my bad.... I need to get used to checking Discord for that sorta thing! My friends and I strictly use it as if it was Mumble or Ventrilo (VoIP) 😅

Anyways, thanks for the links!
